apps/functions/ng-dev/ng-dev-token.ts

```diff
@@ -0,0 +1,57 @@
+import * as functions from 'firebase-functions';
+import * as admin from 'firebase-admin';
+import {CallableContext} from 'firebase-functions/lib/providers/https';
+
+/**
+ * Request a short lived ng-dev token.  If granted, we rely on session cookies as this token.  The token
```

Comment:
The typical nit, but I'm bring it up because you didn't tell me to ignore it 😄 

```suggestion
 * Request a short lived ng-dev token. If granted, we rely on session cookies as this token. The token
```